### PR TITLE
Labs-13 Get toggles activated by user

### PR DESCRIPTION
**Pull request by MeliaBurnley**
This can't be merged until labs-mysql implements the: `MysqlActivatedToggleGateway` but it all works otherwise

### CODE REVIEW

**Comment by Clearbooks-Tom** [comment]
ping @VolodymyrSergeyev

File: composer.json [hunk 1]
```diff
@@ -32,6 +32,14 @@
             "url": "https://github.com/clearbooks/labs-mysql"
         },
         {
+            "type": "vcs",
```
Comment (Clearbooks-Tom) [inline composer.json]: whoooa how come we're depending on the PHP client now?

Comment (MeliaBurnley) [inline composer.json]: https://github.com/composer/composer/issues/1478

Comment (MeliaBurnley) [inline composer.json]: Although that seems to be fine now, who knows

**Comment by vokoblin** [comment]
Pong @Clearbooks-Tom

File: src/Toggle/GetTogglesActivatedByUser.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    /**
+     * GetTogglesActivatedByUser constructor.
+     * @param GetActivatedToggles $getActivatedToggles
+     */
+    public function __construct(GetActivatedToggles $getActivatedToggles)
+    {
+        $this->getActivatedToggles = $getActivatedToggles;
+    }
+
+    public function execute(Request $request)
```
Comment (vokoblin) [inline src/Toggle/GetTogglesActivatedByUser.php:38]: This is missing a DocBlock

**Comment by vokoblin** [comment]
Travis is failing :tongue: but logic wise looks good to me.

**Comment by vokoblin** [comment]
https://github.com/clearbooks/labs-mysql/pull/13 was merged

**Pull request merged**